lib/PuppeteerSharp.Tests/PuppeteerTests/PuppeteerConnectTests.cs

```diff
@@ -111,7 +111,7 @@ public async Task ShouldBeAbleToReconnectToADisconnectedBrowser()
         public async Task ShouldBeAbleToConnectToTheSamePageSimultaneously()
         {
             var browserOne = await Puppeteer.LaunchAsync(new LaunchOptions());
-            var browserTwo = await Puppeteer.ConnectAsync(new ConnectOptions
+            await Puppeteer.ConnectAsync(new ConnectOptions
```

Comment:
This test is wrong. According to this https://github.com/puppeteer/puppeteer/blob/054d782c823c363c418bf478b0dbefa26673b358/test/launcher.spec.ts#L592 we should create a page with browserTwo in line 125.